pkg/collector/corechecks/oracle-dbm/oracle_dictionary.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build oracle
+
+package oracle
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	go_ora "github.com/sijms/go-ora/v2"
+)
+
+func getFullSQLText(c *Check, SQLStatement *string, key string, value string) error {
+	var err error
+	var sql string
+	if c.driver == "godror" {
```

Comment:
Really a minor nit since we're unlikely to need to support many more drivers but this might still look a little nicer as a switch case:
```
switch c.driver {
    case "godror":
   ...
    case "oracle":
   ...
}
```